frontend/integration-tests/tests/deploy-image.scenario.ts

```diff
@@ -24,7 +24,6 @@ describe('Deploy Image', () => {
 
     it('should render applications dropdown disabled', async () => {
       // Navigate to the deploy-image page
-      await browser.get(`${appHost}/deploy-image/ns/${testName}?preselected-ns=${testName}`);
```

Comment:
Why is this removed?
Also, the comment wasn't removed.

Reply:
In the first test, we are already navigating to deploy image form. This was causing the page to refresh again even though we are already on the deploy image form. https://github.com/openshift/console/pull/4474/files#diff-eb6a4820e46e49f73fc3bf67cc643894R16

Reply:
removed the comment